src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1389,6 +1389,206 @@ public async Task TestRuntimeBaseRouteInNextLinkForPaginatedRestResponse()
+
+                // Delete the new record created as part of this test
+                if (entityType is EntitySourceType.Table)
+                {
+                    HttpRequestMessage cleanupRequest = new(HttpMethod.Delete, locationHeader);
+                    await client.SendAsync(cleanupRequest);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Validates the Location header field returned for a POST request when it results in a 201 response. The idea behind returning
+        /// a Location header is to provide a URL against which a GET request can be performed to fetch the details of the new item.
+        /// Base Route is configured in the config file used for this test. So, it is expected that the Location header returned will contain the base-route.
+        /// This test performs a POST request, and checks if it results in a 201 response. If so, the test validates the correctness of the Location header in two steps.
+        /// Since, base-route has significance only in the SWA-DAB integrated scenario and this test is executed against DAB running independently,
+        /// a subsequent GET request against the Location header will result in an error. So, the correctness of the base-route returned is validated with the help of
+        /// an expected location header value. The correctness of the PK part of the Location string is validated by performing a GET request after stripping off
+        /// the base-route from the Location URL.
+        /// </summary>
+        /// <param name="entityType">Type of the entity</param>
+        /// <param name="requestPath">Request path for performing POST API requests on the entity</param>
+        /// <param name="baseRoute">Configured base route</param>
+        /// <param name="expectedLocationHeader">Expected value for Location field in the response header. Since, the PK of the new record is not known beforehand,
+        /// the expectedLocationHeader excludes the PK. Because of this, the actual location header is validated by checking if it starts with the expectedLocationHeader.</param>
+        [DataTestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        [DataRow(EntitySourceType.Table, "/api/Book", "/data-api", "http://localhost/data-api/api/Book/id/", DisplayName = "Location Header validation - Table, Base Route configured")]
+        [DataRow(EntitySourceType.StoredProcedure, "/api/GetBooks", "/data-api", "http://localhost/data-api/api/GetBooks", DisplayName = "Location Header validation - Stored Procedure, Base Route configured")]
+        public async Task ValidateLocationHeaderWhenBaseRouteIsConfigured(EntitySourceType entityType,
+                                                                          string requestPath,
+                                                                          string baseRoute,
+                                                                          string expectedLocationHeader)
```

Comment:
first param should be bumped to next line and then indentation is consistent with other tests that have params.